graylog2-server/src/test/java/org/graylog2/indexer/rotation/TimeBasedRotationStrategyTest.java

```diff
@@ -88,13 +101,16 @@ public void shouldRotateHourly() {
         final InstantMillisProvider clock = new InstantMillisProvider(initialTime);
         DateTimeUtils.setCurrentMillisProvider(clock);
 
+        Indices indices = mock(Indices.class);
+        when(indices.indexCreationDate(anyString())).thenReturn(initialTime.minus(Period.minutes(5)));
+
         final Period period = Period.hours(1);
-        final TimeBasedRotationStrategy hourlyRotation = new TimeBasedRotationStrategy(period);
+        final TimeBasedRotationStrategy hourlyRotation = new TimeBasedRotationStrategy(period, indices);
 
         RotationStrategy.Result result;
 
         result = hourlyRotation.shouldRotate("ignored");
```

Comment:
As a matter of fact the index name is not ignored anymore. :wink:
But let's leave it that way to keep the change set as small as possible.